apps/web/middleware.ts

```
@@ -96,6 +99,25 @@ const middleware = async (req: NextRequest): Promise<NextResponse<unknown>> => {
     res.cookies.delete("next-auth.session-token");
   }
 
+  if (url.pathname.startsWith("/router")) {
+    const userAgent = req.headers.get("user-agent");
+
+    if (!userAgent) {
+      logger.error("Missing user agent in web middleware");
+      return NextResponse.next();
+    }
+
+    try {
+      const isBot = isbot(userAgent);
+      if (isBot) {
+        return NextResponse.status(200);
+      }
+    } catch (error) {
+      logger.error("Error checking if user agent is a bot in web middlware:", error);
+      return NextResponse.next();
```

If there is any error then we should fail silently to the user